main/src/main/java/org/mobilitydata/gtfsvalidator/cli/Arguments.java

```diff
@@ -38,11 +38,11 @@
   private int numThreads = 1;
 
   @Parameter(
-      names = {"-f", "--feed_name"},
```

Comment:
If we remove `-f` completely I believe this will be a breaking change for clients/users and the next release should be v3.